mslice/tests/cli_helper_functions_test.py

```diff
@@ -158,11 +158,16 @@ def test_that_is_slice_works_as_expected(self):
     def test_that_is_cut_works_as_expected(self, is_gui):
         is_gui.return_value = True
         workspace = self.create_workspace('workspace')
+        psd_workspace = self.create_pixel_workspace('psd_workspace')
         cut_ws = Cut(workspace)
+        cut_psd_ws = Cut(psd_workspace)
 
         return_value = is_cut(cut_ws)
         self.assertEqual(return_value, True)
 
+        return_value = is_cut(cut_psd_ws)
+        self.assertEqual(return_value, True)
```

Comment:
It looks good. However, perhaps, we could move code lines related to workspace and psd_workspace to separate blocks (within the same test function).